bench-vortex/src/statpopgen/statpopgen_benchmark.rs

```diff
@@ -0,0 +1,257 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: Copyright the Vortex contributors
+
+use std::fs;
+use std::path::{Path, PathBuf};
+use std::sync::Arc;
+
+use anyhow::Result;
+use datafusion::datasource::file_format::parquet::ParquetFormat;
+use datafusion::datasource::listing::{
+    ListingOptions, ListingTable, ListingTableConfig, ListingTableUrl,
+};
+use datafusion::prelude::SessionContext;
+use tracing::info;
+use url::Url;
+use vortex::error::{VortexResult, vortex_err};
+use vortex_datafusion::VortexFormat;
+
+use crate::benchmark_trait::Benchmark;
+use crate::engines::EngineCtx;
+use crate::statpopgen::schema::SCHEMA;
+use crate::{BenchmarkDataset, Format, Target};
+
+/// Statistical population genetics benchmark implementation.
```

Comment:
Was just thinking whether it'd be nice to have top-level mod.rs docs but this works.